### PR TITLE
Document ocaml-print-intf

### PR DESCRIPTION
**Pull request by emillon**
This adds a FAQ entry about how to generate a mli file for a ml-only module.

### CODE REVIEW

File: doc/faq.rst [hunk 1]
```diff
@@ -98,3 +98,25 @@ build.
 In that case, one can pass ``-j1 --no-buffer`` so that the commands are directly
 printed on the console (and the parallelism is disabled so that the output stays
 readable).
+
+How can I generate a mli file from a ml file?
```
Comment (CraigFe) [inline doc/faq.rst:102]:
```suggestion
How can I generate an mli file from an ml file?
```

(Please forgive the drive-by nit :upside_down_face:)

**Comment by emillon** [comment]
Thanks everyone!
